source/common/runtime/runtime_features.h

```diff
@@ -18,6 +18,9 @@ bool isRuntimeFeature(absl::string_view feature);
 bool runtimeFeatureEnabled(absl::string_view feature);
 uint64_t getInteger(absl::string_view feature, uint64_t default_value);
 
+void runtimeIsInitialized();
+bool isRuntimeInitialized();
```

Comment:
nit: these 2 functions are kind of similar in naming. I might do s/runtimeIsInitialized/markRuntimeInitialized but up to you.